test/unit/jobs/job_conf.sample_advanced.yml

```
@@ -125,6 +125,13 @@ runners:
     # higher value (in seconds) (or `None` to use blocking connections).
     #amqp_consumer_timeout: None
 
+  pulsar_k8s:
+    # Use a two-container Kubernetes to run jobs - one for staging and one
+    # for tool execution.
+    load: galaxy.jobs.runners.pulsar:PulsarKubernetesCoexecutionJobRunner
+    galaxy_url: http://docker.for.mac.localhost:8080
```

Since we figured that out for IEs as well, it seems that the recommendation for modern docker for mac (starting 18.03) is `host.docker.internal` (https://docs.docker.com/docker-for-mac/networking/)
```suggestion
    galaxy_url: http://host.docker.internal:8080
```